[vault-env] divide by zero error

**Describe the bug**:
Versions of vault-env > 1.17 cause a panic when injecting transit-encrypted secrets

```
panic: runtime error: integer divide by zero                                                                                                                                                                          
                                                                                                                                                                                                                      
goroutine 1 [running]:                                                                                                                                                                                                
github.com/banzaicloud/bank-vaults/internal/injector.paginate(...)                                                                                                                                                    
    /build/internal/injector/injector.go:96                                                                                                                                                                           
github.com/banzaicloud/bank-vaults/internal/injector.SecretInjector.preprocessTransitSecrets({{{0xc000046075, 0x1b}, {0xc0000420b3, 0xb}, 0x0, 0x0, 0x0}, 0xc00007ebe0, {0x0, 0x0}, ...}, ...)                        
    /build/internal/injector/injector.go:133 +0xb31                                                                                                                                                                   
github.com/banzaicloud/bank-vaults/internal/injector.SecretInjector.InjectSecretsFromVault({{{0xc000046075, 0x1b}, {0xc0000420b3, 0xb}, 0x0, 0x0, 0x0}, 0xc00007ebe0, {0x0, 0x0}, ...}, ...)                          
    /build/internal/injector/injector.go:168 +0x85                                                                                                                                                                    
main.main()                                                                                                                                                                                                           
    /build/cmd/vault-env/main.go:279 +0x1048   
```

This appears to be due to the paging functionality added [here](https://github.com/banzaicloud/bank-vaults/pull/1697).

It appears that TransitBatchSize is never set when entering from the vault-env main function. 

**Expected behaviour**:
No panics occur

**Steps to reproduce the bug**:
Have a Deployment with a transit encrypted secret in the env variables.
Have secret injection enabled

**Additional context**:
Add any other context about the problem here.

**Environment details**:
- Kubernetes version (e.g. v1.10.2): 1.24
- Cloud-provider/provisioner (e.g. AKS, GKE, EKS, PKE etc): EKS
- bank-vaults version (e.g. 0.4.17): 1.18.0
- Install method (e.g. helm or static manifests): helm
- Logs from the misbehaving component (and any other relevant logs):
- Resource definition (possibly in YAML format) that caused the issue, without sensitive data:

/kind bug

--- a/cmd/vault-env/main.go
+++ b/cmd/vault-env/main.go
@@ -75,6 +75,7 @@ var sanitizeEnvmap = map[string]envType{
 	"VAULT_AUTH_METHOD":            {login: false},
 	"VAULT_TRANSIT_KEY_ID":         {login: false},
 	"VAULT_TRANSIT_PATH":           {login: false},
+	"VAULT_TRANSIT_BATCH_SIZE":     {login: false},
 	"VAULT_IGNORE_MISSING_SECRETS": {login: false},
 	"VAULT_ENV_PASSTHROUGH":        {login: false},
 	"VAULT_JSON_LOG":               {login: false},
@@ -253,6 +254,7 @@ func main() {
 	config := injector.Config{
 		TransitKeyID:         os.Getenv("VAULT_TRANSIT_KEY_ID"),
 		TransitPath:          os.Getenv("VAULT_TRANSIT_PATH"),
+		TransitBatchSize:     cast.ToInt(os.Getenv("VAULT_TRANSIT_BATCH_SIZE")),
 		DaemonMode:           daemonMode,
 		IgnoreMissingSecrets: ignoreMissingSecrets,
 	}

--- a/pkg/webhook/pod.go
+++ b/pkg/webhook/pod.go
@@ -375,20 +375,30 @@ func (mw *MutatingWebhook) mutateContainers(ctx context.Context, containers []co
 				},
 			}...)
 		}
-		if len(vaultConfig.VaultNamespace) > 0 {
-			container.Env = append(container.Env, []corev1.EnvVar{
-				{
-					Name:  "VAULT_NAMESPACE",
-					Value: vaultConfig.VaultNamespace,
-				},
-			}...)
-		}
 
 		if len(vaultConfig.TransitPath) > 0 {
 			container.Env = append(container.Env, []corev1.EnvVar{
 				{
 					Name:  "VAULT_TRANSIT_PATH",
 					Value: vaultConfig.TransitPath,
+				},
+			}...)
+		}
+
+		if vaultConfig.TransitBatchSize > 0 {
+			container.Env = append(container.Env, []corev1.EnvVar{
+				{
+					Name:  "VAULT_TRANSIT_BATCH_SIZE",
+					Value: strconv.Itoa(vaultConfig.TransitBatchSize),
+				},
+			}...)
+		}
+
+		if len(vaultConfig.VaultNamespace) > 0 {
+			container.Env = append(container.Env, []corev1.EnvVar{
+				{
+					Name:  "VAULT_NAMESPACE",
+					Value: vaultConfig.VaultNamespace,
 				},
 			}...)
 		}

